### PR TITLE
Add description field to test-defs

### PR DESCRIPTION
**Pull request by tobiscr**
**Description**
Adding description field to test definitions.

**Related issue(s)**
Agreed in Kyma quality SIG group.

### CODE REVIEW

**Review by PK85 — DISMISSED** [review]

File: docs/crd-test-definition.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.template** |    **YES**   | Describes the Pod that will be created. This field is of `PodTemplateSpec` type from the Kubernetes API. Find its detailed description [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#podtemplatespec-v1-core)  |
 | **spec.skip**     |    **NO**    | Indicates that a test should not be executed. The default value is `false`. This feature is not yet implemented. |
 | **spec.disableConcurrency** | **NO** | Disallows running the given test concurrently. The default value is `false`. 
-| **spec.timeout** | **NO** | Defines the maximal duration of a test, after which it is terminated and marked as **Failed**. This feature is not yet implemented. 
+| **spec.timeout** | **NO** | Defines the maximal duration of a test, after which it is terminated and marked as **Failed**. This feature is not yet implemented.
+| **spec.description** | **NO** | Describes the test case in detail (e.g. scope, test scenario, edge cases, known limitations etc.).
```
Comment (klaudiagrz) [inline docs/crd-test-definition.md]:
```suggestion
| **spec.description** | **NO** | Describes the details of the test case, such as the scope, the test scenario, edge cases, known limitations, etc.
```

**Review by klaudiagrz — APPROVED** [review]

**Review by polskikiel — APPROVED** [review]
